src/pages/fill-the-form-job/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { IJobSeekerProps } from '@allTypes/reduxTypes/areaSpecializationTypes'
+import { AreaOfSpecialization } from '@components/AreaOfSpecialization'
+import { Container } from '@components/Container'
+import { FilTheFormJobReview } from '@components/FilTheFormJobReview'
+import { LeftIcon } from '@components/Icons/LeftIcon'
+import { Translation } from '@constants/translations'
+import { Tab } from '@headlessui/react'
+import { Button } from '@uiComponents/Button'
+
+const FillTheForm = () => {
+  const [t] = useTranslation()
+
+  const [selectedIndex, setSelectedIndex] = useState<number>(0)
+  const [jobSeekerReview, setJobSeekerReview] = useState<null | IJobSeekerProps>(null)
```

Comment:
Not sure why we need this.
Any data that need global usage should be kept in hook form

Reply:
done